src/command/stamp/show.ts

```diff
@@ -0,0 +1,23 @@
+import { Argument, LeafCommand } from 'furious-commander'
+import { StampCommand } from './stamp-command'
+
+export class Show extends StampCommand implements LeafCommand {
+  public readonly name = 'show'
+
+  public readonly aliases = []
+
+  public readonly description = 'Show a specific postage stamp'
+
+  @Argument({ key: 'batch-id', description: 'Batch ID of the postage stamp', required: true })
+  public batchId!: string
```

Comment:
Other commands simply call this `stamp`, naming should be more consistent.

Reply:
This came up with the `bee-js` too. The root cause is that `batch` and `stamp` is both used to indicate the same thing in Bee and we sticked to `postageBatch` in the JS API.

However I find `stamp` a bit more specific and understandable as a user facing concept, so I would use that (or maybe `stamp-id` as argument). This may create a bit confusion in the code, so it would be good to add a comment about it.

Reply:
Pushed a followup for this just now, without seeing your comment.

I ended up using:

```
key: 'stamp',
description: 'Batch ID of the postage stamp to use'
```

Maybe `ID of the postage batch to use` is a better description based on your feedback? But then `stamp` stands out a bit, not being mentioned in the description at all. Wording is hard. 🙂

Reply:
I would go with `ID of the postage stamp to use`